src/docs/Foundations/Layout/Container/code.mdx

```diff
@@ -29,4 +33,6 @@ If you want to use only the default paddings, but still want the container to be
 </main>
 ```
 
```

Comment:
sentence change for `fluid container` part:

Use the ml-container--fluid modifiers if you want to use the default paddings only but still want the container to be fluid at any viewport width.

Reply:
fixed